components/elements/input.js

```diff
@@ -77,16 +76,11 @@ export function FormInput({
         )}
       </div>
     </div>
-  );
+  )
 }
 
-export function PasswordInput({
-  setValue,
-  className,
-  password = false,
-  ...props
-}) {
-  const [type, setType] = useState("password");
+export function PasswordInput({ setValue, className, ...props }) {
```

Comment:
I would suggest to revise elements and remove unused code from `input.js` for example.